sajibportfolio.html

```diff
@@ -1,273 +1,321 @@
-              </li>
-            </ul>
-
-          
-       
-      </li>
-      <li>
-        <div id="spec_name">
-          <p class="lfs">Frameworks</p>
-          <div><img src="statics/images/select.png" alt="select"></div>
-        </div>
-        <div class="lfsi">
-          <hr />
-        </div>
-      </li>
-      <li>
-        <div id="spec_name">
-          <p class="lfs">Skills</p>
-          <div><img src="statics/images/select.png" alt="select"></div>
-        </div>
-        <div class="lhr" id="l2">
-          <hr />
-        </div>
-      </li>
-    </ul>
-  </div>
+  
+      <div id="my_skills">
+  
+      <ul class="ss">
+        <li>  
+              <div id="spec_name">
+                <p class="lfs">Languages</p>
+                <img class="lfsi" src="statics/images/fleche_dessous.png">
+              </div>
+  
+              <ul id="programmes">
+                <li class="pro">
+                  <img class="img1" src="statics/images/java.png" alt="javascript image">
+                  <p class="lng1">Javascript</p>
+                </li>
+                <li class="pro">
+                  <img class="img1" src="statics/images/html.png" alt="html image">
+                  <p>HTML</p>
+                </li>
+                <li class="pro">
+                  <img class="img1" src="statics/images/css.png" alt="css image">
+                  <p>CSS</p>
+                </li>
+              </ul>
+  
+            
+         
+        </li>
+        <li>
+          <div id="spec_name">
+            <p class="lfs">Frameworks</p>
+            <div><img src="statics/images/select.png" alt="select"></div>
+          </div>
+          <div class="lfsi">
+            <hr />
+          </div>
+        </li>
+        <li>
+          <div id="spec_name">
+            <p class="lfs">Skills</p>
+            <div><img src="statics/images/select.png" alt="select"></div>
+          </div>
+          <div class="lhr" id="l2">
+            <hr />
+          </div>
+        </li>
+      </ul>
     </div>
-  </section>
-</body>
+      </div>
+    </section>
+    <section class="form-section" id="contact_me">
+      
+        <h2 class="contact-title">Contact me</h2>
+        <p class="contact-desc">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class="form-container" action="https://formspree.io/f/myyvrplv" method="post">
+          <input type="text" name="username" placeholder="Yeremias" maxlength="30" required>
+          <input class="form-email" type="email" name="email" placeholder="natajayanj14@gmail.com" required>
+          <textarea class="form-comment" name="message" placeholder="Write your message here" row="10" maxlength="500" required></textarea>
+          <button type="submit" name="submit">Get in touch</button>
+          <span></span>
+          <span></span>
+        </form>
```

Comment:
- [Optional] Dear @sajibarafatsiddiqui, Although you did great with the `css` in your form till your design is not as close as possible to the `figma` design. This is among the main requirement and I'm making it optional only because it has not been mentioned to you to update your form section before.

### Your design 
![Screenshot 2022-12-07 203111](https://user-images.githubusercontent.com/69077061/206252362-5cd6cfa2-c941-4149-8f74-6e3982e100c4.png)

### Figma design 
![Screenshot 2022-12-07 203211](https://user-images.githubusercontent.com/69077061/206252370-ef34a15a-510a-45c3-a166-f2bff738bbd9.png)

Reply:
resolved it